README.md

```diff
@@ -17,7 +17,7 @@ As with all ExAws services, you'll need a compatible HTTP client (defaults to `:
 ```elixir
 defp deps do
   [
-    {:ex_aws, "~> 2.0"},
+    {:ex_aws, "~> 2.1"},
```

Comment:
This this is probably copy and pasted often enough, it might as well point to the latest release.